teps/0128-scheduled-runs.md

```diff
@@ -63,7 +64,100 @@ If the EventListener is exposed outside of the cluster, the user must also sign
 - Release Pipeline: User would like to create releases of their project on a nightly basis.
 
 ## Proposal
+Create ScheduledRun CRD with `resourcetemplates` and `schedule` field
 
+This solution doesn't use TriggerTemplate or TriggerBindings. Instead, it's spec has `resourcetemplates` which can have one or more pipelineruns, taskruns or other allowed Kubernetes and Tekton resources.
+
+```yaml
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: ScheduledRun
```

Comment:
nit: for naming, since we're using resourcetemplates generically, does it make sense to call this a scheduled run as opposed to a scheduled trigger or something else?

Reply:
Yes, maybe we can name it as scheduledTemplate.